### PR TITLE
6pm 1 drop down menu (and zoom room link optional)

### PR DESCRIPTION
**Pull request by moisesgtz**
Working drop-down menu for the days of the week on the "Add New Office Hours" Page.

### CODE REVIEW

**Comment by moisesgtz** [comment]
Added zoom link to be optional for creating office hours

File: javascript/src/main/components/OfficeHours/OfficeHourForm.js [hunk 1]
```diff
@@ -50,10 +51,6 @@ const OfficeHourForm = ({ createOfficeHour, /*updateOfficeHour, /*existingOffice
         validList.push(endTimeValid); 
 
         // check day of week
```
Comment (pconrad) [inline javascript/src/main/components/OfficeHours/OfficeHourForm.js:53]: Can probably remove this comment since the code has been removed.

**Review by pconrad — CHANGES_REQUESTED** [review]
One small comment to remove.  Otherwise, LGTM

**Comment by pconrad** [comment]
20 pts for the drop down, 10 points for making zoom room link optional.

**Review by pconrad — APPROVED** [review]
LGTM

**Review by LeoJr99 — APPROVED** [review]
LGTM

**Pull request merged**